GUI/GUI.csproj

```diff
@@ -13,6 +13,7 @@
     <GarbageCollectionAdaptationMode>1</GarbageCollectionAdaptationMode>
     <StartupObject>GUI.Program</StartupObject>
     <AssemblyName>Source2Viewer</AssemblyName>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
```

Comment:
can you remove this, if its for the dll imports, theres a way to do them without unsafe (yes i know there's a warning that complains about it, disable it)

Reply:
got it